samples/bluetooth/bap_broadcast_source/Kconfig

```
@@ -45,4 +45,18 @@ config BROADCAST_CODE
 	   Setting a non-empty string for this option will encrypt the broadcast using this
 	   string as the broadcast code. The length of the string shall be between 1 and 16 octets.
 
+config STATIC_BROADCAST_ID
+	bool "Use static broadcast id"
+	default n
+	help
```

```suggestion
	bool "Use static broadcast id"
	help
```

Kconfig bools default to `n`